docs/products.md

```diff
@@ -21,92 +21,116 @@ Welcome to the **OPERA** products documentation page. This section provides deta
 
-| Product Name         | Ancillaries | ATBD | Product Spec                                             | ASF Landing Page Link                    |
-| -------------------- | ----------- | ---- | -------------------------------------------------------- | ---------------------------------------- |
-| RTC-S1               | [RTC S1 Bursts][RTC S1 Bursts]            |      | [RTC-S1 Product Spec][RTC-S1 Product Spec]               | [OPERA_L2_RTC-S1_V1][rtc_s1_data]        |
-| RTC-S1 Static Layers | [RTC S1 Bursts][RTC S1 Bursts]            |      | [RTC-S1-STATIC Product Spec][RTC-S1-STATIC Product Spec] | [OPERA_L2_RTC-S1-STATIC_V1][rtc_s1_static_data] |
+| Product Name         | ATBD | Product Spec                                             | ASF Landing Page Link                           |
+| -------------------- |------|----------------------------------------------------------|-------------------------------------------------|
+| RTC-S1               |      | [RTC-S1 Product Spec][RTC-S1 Product Spec]               | [OPERA_L2_RTC-S1_V1][rtc_s1_data]               |
+| RTC-S1 Static Layers |      | [RTC-S1-STATIC Product Spec][RTC-S1-STATIC Product Spec] | [OPERA_L2_RTC-S1-STATIC_V1][rtc_s1_static_data] |
 
 ## Coregistered Single-Look Complex (CSLC) Products
 
-Product releases for the Coregistered Single-look Complex product suite.  Currently these products are only created from Sentinel-1 data (CSLC-S1).  You can find more information about these products here: [CSLC Product Suite](https://www.jpl.nasa.gov/go/opera/products/cslc-product-suite)
+Product releases for the Coregistered Single-look Complex product suite.  Currently, these products are only created from Sentinel-1 data (CSLC-S1).  You can find more information about these products here: [CSLC Product Suite](https://www.jpl.nasa.gov/go/opera/products/cslc-product-suite)
 
-| Product Name | Product Version | Release Date | SAS Version                | PGE Version                                                            |
-| ------------ | --------------- | ------------ | -------------------------- | -------------------------------------------------------------------- |
-| CSLC-S1      | 1.1             | 2024-04-01   | [0.5.5][cslc_s1_sas_0.5.5] | [2.1.1 (zip)][cslc_s1_pge_2.1.1] |
-| CSLC-S1      | 1.0             | 2023-10-04   | [0.5.4][cslc_s1_sas_0.5.4] | [2.1.0 (zip)][cslc_s1_pge_2.1.0]                                           |
+| Product Name | Product Version | Release Date | SAS Version                | PGE Version                                                                 |
+| ------------ | --------------- | ------------ | -------------------------- |-----------------------------------------------------------------------------|
+| CSLC-S1      | 1.1             | 2024-04-01   | [0.5.5][cslc_s1_sas_0.5.5] | [2.1.3 (docker)][cslc_s1_pge_docker_2.1.3] [2.1.3 (zip)][cslc_s1_pge_2.1.3] |
```

Comment:
@seongsujeong mentioned that there has been a release of CSLC-S1 v.0.5.6. Assuming we're not using that yet in our latest PGE?

Reply:
The CSLC PGE v2.1.3 corresponds to the SAS v0.5.6